### PR TITLE
Release notes for SDK4.0.0

### PR DESCRIPTION
**Pull request by varkumaradobe**
## Description

Release notes for SDK4.0.0

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/overview/releasenotes.md [hunk 1]
```diff
@@ -173,6 +173,18 @@ Upgrading to the latest SDK should not break existing applications.
 
 ## Change history
 
+### 4.0.0 (January, 2024; major release)
+| Change  | Language | Description                                                                                   |
+|---------|----------| --------------------------------------------------------------------------------------------- |
+| Changed | Java     | Released SDK v4.0.0, introducing new interfaces fully leveraging power of new PDF Services rest APIs.|
```
Comment (adityaganotra) [inline src/pages/overview/releasenotes.md:179]: Released Adobe PDF Services Java SDK v4.0.0 ...

File: src/pages/overview/releasenotes.md [hunk 2]
```diff
@@ -173,6 +173,18 @@ Upgrading to the latest SDK should not break existing applications.
...
+| Changed | Java     | Updated PDF Properties operation to return PDFProperties object along with JSON string. |
+| Changed | Java     | Updated Extract PDF operation to return content and resourse asset along with content JSON object.|
+| New     | Java     | Added Support for delete asset.|
```
Comment (adityaganotra) [inline src/pages/overview/releasenotes.md:182]: Added support ...

File: src/pages/overview/releasenotes.md [hunk 3]
```diff
@@ -173,6 +173,18 @@ Upgrading to the latest SDK should not break existing applications.
...
+| New     | Java     | Added Support for refresh download URI.|
+| New     | Java     | Added Support for callback notifiers for effiecient polling.|
```
Comment (adityaganotra) [inline src/pages/overview/releasenotes.md:184]: Can we club all the Added support features.

**Review by adityaganotra — CHANGES_REQUESTED** [review]

**Review by tanmaysharma21 — APPROVED** [review]
webhook notifier change seems good to me.

File: src/pages/overview/releasenotes.md (hunk 2 above)
Comment (varkumaradobe) [reply src/pages/overview/releasenotes.md]: Done

File: src/pages/overview/releasenotes.md (hunk 3 above)
Comment (varkumaradobe) [reply src/pages/overview/releasenotes.md]: Done

File: src/pages/overview/releasenotes.md (hunk 1 above)
Comment (varkumaradobe) [reply src/pages/overview/releasenotes.md]: Done

File: src/pages/overview/releasenotes.md (hunk 2 above)
Comment (varkumaradobe) [reply src/pages/overview/releasenotes.md]: Done

**Review by adityaganotra — APPROVED** [review]

**Review by adityaganotra — APPROVED** [review]
LGTM

**Review by Prerit12 — APPROVED** [review]
Language is looking good to me. Ordering can be confirmed.

**Pull request merged**